hypershift-operator/controllers/hostedcluster/hostedcluster_controller_test.go

```diff
@@ -275,3 +277,90 @@ func TestComputeHostedClusterAvailability(t *testing.T) {
 		})
 	}
 }
+
+func listIncludesString(list []string, target string) bool {
```

Comment:
This function can be deleted and replaced by using `sets.String` which gives you nice set semantics

Reply:
thanks! i haven't used that before, i'll check it out.